src/server/svr_jobfunc.c

```diff
@@ -5964,6 +5964,86 @@ send_job_exec_update_to_mom(job *pjob, char *err_msg, int err_msg_sz,
 	return (rc);
 }
 
+/**
+ *
+ * @brief
+ *	Extracts mom hostnames from exechostx and adds it to list
+ *
+ * @param[in/out]	to_head - destination reliable_job_node list
+ * @param[in]	exechostx - string in exechost format
+ *
+ * @return 	void
+ */
+static void
+populate_mom_list(pbs_list_head *to_head, char *exechostx)
+{
+	char *hostn = NULL, *last = NULL, *peh;
+	int  hasprn = 0;
+
+	if ((to_head == NULL) || (exechostx == NULL) || !*exechostx)
```

Comment:
We should be consistent with our checks. Either...
```
if ((to_head == NULL) || (exechostx == NULL) || (*exechostx != '\0'))
```
or...
```
if (!to_head || !exechostx || !*exechostx)
```
I'm partial to the latter.

Reply:
OK, handled them in commit https://github.com/PBSPro/pbspro/pull/1452/commits/8f309272ad00f7587c830cad07238cb40598e24a